openml-scikit/README.md

```diff
@@ -39,3 +39,7 @@ When running the java process that will use this provider, be sure to have confi
 ```bash
 export JAVA_LIBRARY_PATH=/usr/local/anaconda3/envs/scikit-env-test/lib
 ```
+
+### Usage
+
+When the user imports a model to the Feedzai platform using this provider, the import assumes a folder called ```model```, containing a pickle file (.pkl).
```

Comment:
I usually like to give examples like Unix "tree" command.
E.g.: 

(don't use this *literal* example :) )

`nuno.diegues@Laptop-511:~/workspace/feedzai-openml-python/openml-scikit/src/test/resources$ tree
.
├── no_probability
│   └── model
│       └── svc_no_proba.joblib.pkl
├── rf
│   └── model
│       └── svc__rf_proba.joblib.pkl
└── svm
    ├── model
    │   └── svc.joblib.pkl
    └── model.json
`

Reply:
Well, the indentation is not working great, so let's try with a screenshot:
![image](https://user-images.githubusercontent.com/889015/41847157-ef244538-7870-11e8-8b37-41661d761304.png)